tools/echo_server/readme.md

```diff
@@ -41,6 +41,10 @@ The JSON file contains the following options:
     1. The repeat interval in seconds. When in repeat mode, the echo server will wait for this amount of time until the next echoing.
 1. server-port
     1. Specify which port to open a socket on.
+1. server-certificate
+    1. Server certificate in X509 format. When both `server-certificate` and `server-certificate-location` are specified, this option will be used instead.
+1. server-key
+    1. Server private key in X509 format. When both `server-key` and `server-key-location` are specified, this option will be used instead.
```

Comment:
Nit but the key will not be in X509, as it is a standard only related to certificates

Reply:
fixed in next rev.